vehicle_model:
    centreofgravity_to_frontaxle: 0.105
    steering_limits: 0.7
    wheelbase: 0.210
    mass: 3.0
    moment_of_inertia_z: 0.3
    aerodynamic_coefficient: 2.0
    coefficient_of_resistance: 0.01

dynamic_model:
    update_frequency: 10.0
    velocity_threshold: 0.01